File: 250324/학점 계산기/credit-calculator.cpp
#include <iostream>
using namespace std;

int main()
{
    ios::sync_with_stdio(false);
    cin.tie(0);
    cout.tie(0);

    int n = 0;
    cin >> n;
    double avg = 0, sum = 0;
    double score[5] = {0};
    for(int i = 0; i < n; i++) {
        cin >> score[i];
        sum += score[i];
    }
    avg = (double)sum / n;
    cout << fixed;
    cout.precision(1);
    cout << avg << '\n';
    if(avg >= 4.0) {
        cout << "Perfect";
    }
    else if(avg >= 3.0) {
        cout << "Good";
    }
    else {
        cout << "Poor";
    }
    return 0;
}

File: 250323/학점 계산기/credit-calculator.java
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);

        int n = sc.nextInt();
        double[] score = new double[5];
        double sum = 0;
        double avg = 0;
        for(int i = 0; i < n; i++) {
            score[i] = sc.nextDouble();
            sum += score[i];
        }
        avg = sum / n;
        System.out.printf("%.1f\n", avg);
        if(avg >= 4.0) {
            System.out.println("Perfect");
        }
        else if(avg >= 3.0) {
            System.out.println("Good");
        }
        else {
            System.out.println("Poor");
        }
        sc.close();
    }
}